web_src/css/base.css

```diff
@@ -653,6 +653,11 @@ a.label,
   color: var(--color-text);
 }
 
+.ui.secondary.menu {
+  margin-left: 0;
+  margin-right: 0;
+}
+
```

Comment:
I do not think it's safe to do so (for a general style)

IIRC, the Fomantic uses margin/padding tricks to align container/children elements. The margins are necessary for using Fomantic components correctly.

Reply:
I thought so too, but after some testing, I could not notice any negative effects of doing this.